src/core/api/thread_ftd_api.cpp

```diff
@@ -242,6 +249,14 @@ ThreadError otThreadSetSteeringData(otInstance *aInstance, otExtAddress *aExtAdd
 {
     return aInstance->mThreadNetif.GetMle().SetSteeringData(aExtAddress);
 }
+#else
+ThreadError otThreadSetSteeringData(otInstance *aInstance, otExtAddress *aExtAddress)
+{
+    (void)aInstance;
+    (void)aExtAddress;
+
+    return kThreadError_NotCapable;
```

Comment:
I understand `NotCapable` is not really intended for this (I think I may have used it for dynamic log level) but I don't like `NotImplemented` here.
How about if we add a new error type `NotSupported` (suggestion for a better name are vert welcome):
- `NotImplemented` to mean no code exist for this feature/functionality
-`NotSupported` to mean code exists for this, but current build-time config settings disable this
-`NoCapable` to mean something that current (possibly run-time changable) settings won't allow.

Thoughts?